README.md

```diff
@@ -25,44 +25,43 @@ Small and Medium business accounting and ERP
  6. [License](#license)
 
 
-# DESCRIPTION
+# Description
 
-LedgerSMB is a free web-based double-entry accounting system, featuring
-quotation, ordering, invoicing, projects, timecards, inventory management,
-shipping and more ...
+LedgerSMB is a free and interactive web-based double-entry accounting system, featuring
+quotations, ordering, invoicing, projects, timecards, inventory management,
+shipping, and more...
 
 Directly send orders and invoices from the built-in e-mail function to your
-customers or RFQs (request for quotation) to your vendors with PDF attachments,
-from anywhere in the world with the browser-based UI.
+customers or RFQs (Request For Quotation) to your vendors, complete with PDF attachments,
+from anywhere in the world with an easy-to-use browser-based UI.
 
-With its data stored in the enterprise-strength `PostgreSQL` open source
+With its data stored in the enterprise-strength `PostgreSQL` open source relational
 database system, the system is known to operate smoothly for businesses with
 thousands of transactions per week.
 
-Customer visible output is fully customizable in templates, allowing easy and
-fast customization. Supported output formats are PDF, CSV, HTML, ODF and more.
+Customer-visible output is fully customizable in templates, allowing easy and
+fast customization.  Supported output formats include PDF, CSV, HTML, ODF, and more.
 
 
 # System requirements
 
 Note that these are the system requirements for LedgerSMB 1.12.0-dev, the current
-development version. Please check the system requirements for [the 1.11 stable
+development version.  Please check the system requirements for [the 1.11 stable
 version](https://github.com/ledgersmb/LedgerSMB/tree/1.11#system-requirements)
 and [the 1.10 'old stable' version](https://github.com/ledgersmb/LedgerSMB/tree/1.10#system-requirements).
 
 ## Server
 
 * `Perl 5.32+`
 * `PostgreSQL 13+`
-* Web server (e.g. `nginx`, `Apache`, `lighttpd`, `Varnish`)
+* Web server (e.g., `nginx`, `Apache HTTPd`, `lighttpd`, `Varnish`)
```

Comment:
No comma after e.g. (exempli gratia)

Reply:
Normally "e.g." is followed by a comma, and here's a list of 6 well-known guides about English that promote this:

1. Guide to Canadian English Usage, second edition (2007; from Margery Fee & Janice McAlpine):  "Most guides recommend that 'i.e.' and 'e.g.' be preceded and followed by commas" 
2. Chicago Manual of Style:  "A comma is usually used"
3. Blue Book of Grammar and Punctuation:  "Commas are preferable but optional"
4. The Columbia Guide to Standard American English:  "Editors require a comma"
5. The Guide to Grammar and Writing:  "The commas make good sense"
6. Lynch Guide to Grammar:  "Should be followed by a comma"

Reply:
Go then